docs/development/configuration.md

```diff
@@ -17,7 +17,7 @@ Options which have `"globalOnly": true` are reserved only for bot global configu
 
 ### Configuration File
 
-You can override default configuration using a configuration file, with default name `config.js` in the working directory.
+You can override default configuration using a configuration file, with default name `config.js` or `config.cjs` (in ESM repositories) in the working directory.
```

Comment:
Does Renovate find a `config.cjs` automatically or does it need to be specified using RENOVATE_CONFIG_FILE?

Reply:
good question - will find out.

Reply:
Only if specified via `RENOVATE_CONFIG_FILE`

Reply:
In that case the description needs changing so that people don't think it's automatic. Only config.js is automatic

Reply:
remove from docs, just mention that you can use `.cjs` extension with `RENOVATE_CONFIG_FILE` setting

Reply:
done. please let me know if there's anything else I've missed